metrics/metrics.go

```diff
@@ -143,26 +143,34 @@ func CollectProcessMetrics(refresh time.Duration) {
 
 	// Define the various metrics to collect
 	var (
-		cpuSysLoad            = GetOrRegisterGauge("system/cpu/sysload", DefaultRegistry)
-		cpuSysWait            = GetOrRegisterGauge("system/cpu/syswait", DefaultRegistry)
-		cpuProcLoad           = GetOrRegisterGauge("system/cpu/procload", DefaultRegistry)
-		cpuThreads            = GetOrRegisterGauge("system/cpu/threads", DefaultRegistry)
-		cpuGoroutines         = GetOrRegisterGauge("system/cpu/goroutines", DefaultRegistry)
-		cpuSchedLatency       = getOrRegisterRuntimeHistogram("system/cpu/schedlatency", secondsToNs, nil)
-		memPauses             = getOrRegisterRuntimeHistogram("system/memory/pauses", secondsToNs, nil)
-		memAllocs             = GetOrRegisterMeter("system/memory/allocs", DefaultRegistry)
-		memFrees              = GetOrRegisterMeter("system/memory/frees", DefaultRegistry)
-		memTotal              = GetOrRegisterGauge("system/memory/held", DefaultRegistry)
-		heapUsed              = GetOrRegisterGauge("system/memory/used", DefaultRegistry)
-		heapObjects           = GetOrRegisterGauge("system/memory/objects", DefaultRegistry)
-		diskReads             = GetOrRegisterMeter("system/disk/readcount", DefaultRegistry)
-		diskReadBytes         = GetOrRegisterMeter("system/disk/readdata", DefaultRegistry)
-		diskReadBytesCounter  = GetOrRegisterCounter("system/disk/readbytes", DefaultRegistry)
-		diskWrites            = GetOrRegisterMeter("system/disk/writecount", DefaultRegistry)
-		diskWriteBytes        = GetOrRegisterMeter("system/disk/writedata", DefaultRegistry)
-		diskWriteBytesCounter = GetOrRegisterCounter("system/disk/writebytes", DefaultRegistry)
+		cpuSysLoad               = GetOrRegisterGauge("system/cpu/sysload", DefaultRegistry)
+		cpuSysWait               = GetOrRegisterGauge("system/cpu/syswait", DefaultRegistry)
+		cpuProcLoad              = GetOrRegisterGauge("system/cpu/procload", DefaultRegistry)
+		cpuSysSecondsCounter     = GetOrRegisterCounter("system/cpu/seconds/count", DefaultRegistry)
```

Comment:
perhaps 

cpuSysLoadTotal = GetOrRegisterGauge("system/cpu/sysload/total", DefaultRegistry)
cpuSysWaitTotal = GetOrRegisterGauge("system/cpu/syswait/total", DefaultRegistry)
cpuProcLoadTotal = GetOrRegisterGauge("system/cpu/procload/total", DefaultRegistry)

Reply:
Happy to change that.  See new commit.